tensorflow/compiler/mlir/tosa/transforms/legalize_tfl.cc

```diff
@@ -3366,25 +3432,23 @@ LogicalResult ConvertTFLLeakyReluOp::matchAndRewrite(
     return rewriter.notifyMatchFailure(op,
                                        "input or output is not a ShapedType");
 
+  bool input_is_qtype =
+      input_type.getElementType().isa<mlir::quant::UniformQuantizedType>();
   bool output_is_qtype =
       output_type.getElementType().isa<mlir::quant::UniformQuantizedType>();
 
+  if (input_is_qtype != output_is_qtype) {
+    return rewriter.notifyMatchFailure(
+        op,
+        "ConvertTFLLeakyReluOp: input/output tensor should be all quantized or "
```

Comment:
Lets follow the style of the others here, s/ConvertTFLLeakyReluOp: // and remove trailing period

Reply:
done